app/nexus/internal/env/recovery.go

```diff
@@ -0,0 +1,25 @@
+//    \\ SPIKE: Secure your secrets with SPIFFE.
+//  \\\\\ Copyright 2024-present SPIKE contributors.
+// \\\\\\\ SPDX-License-Identifier: Apache-2.0
+
+package env
+
+import (
+	"os"
+	"time"
+)
+
+// RecoveryOperationTimeout returns the recovery timeout duration.
+// It reads from the SPIKE_NEXUS_RECOVERY_TIMEOUT environment variable
+//
+// If the environment variable is not set or is not a valid duration string
+// then it defaults to `0` (no timeout limit).
+func RecoveryOperationTimeout() time.Duration {
+	e := os.Getenv("SPIKE_NEXUS_RECOVERY_TIMEOUT")
```

Comment:
You'll need to add this to the "configuration" section of the documentation too.

https://spike.ist/getting-started/configuration/

Here: https://github.com/spiffe/spike/blob/main/docs-src/content/getting-started/configuration.md